CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Unreleased
 
+## 0.4.0 (03-12-2021)
+
+### Changes
+
+- Changed environment variable prefix from `SPLK_` to `SPLUNK_`. All environment
```

Comment:
This should trigger a major version bump right? It's only a minor bump because of leading 0 right?

Reply:
Correct. It should if we were 1.x but since we are still "unstable", we are allowed to break people's apps unfortunately for them :)

Reply:
I'd have added backward compatibility for this but I know only one customer is using this right now and I can communicate this change to them directly :)